apps/portal/src/app/README.md

```
@@ -0,0 +1,38 @@
+# Project Setup
+
+### 1. Create Local PostgreSQL Database
```

Markdownlint

Expected: h2; Actual: h3
Heading levels should only increment by one level at a time

(MD001, heading-increment)